libmamba/include/mamba/specs/build_number_spec.hpp

```diff
@@ -125,6 +125,7 @@ namespace mamba::specs
          */
         [[nodiscard]] auto contains(BuildNumber point) const -> bool;
 
+        // TODO: only use the `= default` implementation of `operator==` when we will use C++20.
```

Comment:
```suggestion
        // TODO: replace by `= default` implementation of `operator==` when we will use C++20.
```
Same suggestion for all others.